examples/android/app/src/main/jni/main.cpp

```diff
@@ -26,53 +26,53 @@
 int vibrate(sf::Time duration)
 {
     // First we'll need the native activity handle
-    ANativeActivity* activity = sf::getNativeActivity();
+    ANativeActivity& activity = *sf::getNativeActivity();
 
     // Retrieve the JVM and JNI environment
-    JavaVM* vm  = activity->vm;
-    JNIEnv* env = activity->env;
+    JavaVM& vm  = activity.vm;
```

Comment:
I don't think this compiles. `vm` is still a pointer.
For reference: [ANativeActivity structure reference](https://developer.android.com/ndk/reference/struct/a-native-activity#struct_a_native_activity_1a71db04679afeb8c87ba18d398a7f9187)

Reply:
Oh I see. I assumed this file was being compiled in CI but I don't believe it is.